### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by roukmoute**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
 
-Contributions are very welcome!
+Hi there! We're thrilled that you'd like to contribute to this project.
+Your help is essential for keeping it great.
 
-## Running the test suite
+## Submitting a pull request
 
-Install the development dependencies:
+1. [Fork](https://github.com/algolia/algoliasearch-client-php/fork) and clone the repository
```
Comment (greg0ire) [inline CONTRIBUTING.md]: Wat

File: composer.json [hunk 2]
```diff
@@ -20,6 +20,7 @@
     },
     "require-dev": {
         "matthiasnoback/symfony-dependency-injection-test": "^2.0",
+        "phpunit/phpunit": "^6.5",
```
Comment (greg0ire) [inline composer.json]: I use simple-phpunit IIRC

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
+2. Configure and install the dependencies: `composer install`
+3. Make sure the tests pass on your machine via the section [Running the tests](#running-the-tests)
+4. Create a new branch: `git checkout -b my-branch-name`
+5. Make your change, add tests, and make sure the tests still pass
+6. Push to your fork and [submit a pull request](https://github.com/greg0ire/enum/compare)
+7. Pat your self on the back and wait for your pull request to be reviewed and merged.
```
Comment (greg0ire) [inline CONTRIBUTING.md]: Yourself?

File: composer.json (hunk 2 above)
Comment (roukmoute) [reply composer.json]: It does not work because phpunit missing.

Comment (roukmoute) [reply composer.json]: I've just checked:

```bash
▶ php ./vendor/bin/simple-phpunit 
PHP Fatal error:  Uncaught Exception: simple-phpunit requires the "zip" PHP extension to be installed and enabled in order to uncompress the downloaded PHPUnit packages. in /home/roukmoute/workspace/enum/vendor/symfony/phpunit-bridge/bin/simple-phpunit:73
Stack trace:
#0 {main}
  thrown in /home/roukmoute/workspace/enum/vendor/symfony/phpunit-bridge/bin/simple-phpunit on line 73
```

Comment (greg0ire) [reply composer.json]: What does `php -m|grep zip`return?

Comment (roukmoute) [reply composer.json]: empty

Comment (greg0ire) [reply composer.json]: That means the issue is with your system. Maybe it is installed but still not enabled?

Comment (roukmoute) [reply composer.json]: I don't think it currently available with php 7.2.

Comment (greg0ire) [reply composer.json]: Aren't there Travis builds that use 7.2, simple-phpunit and work on this repo?

**Comment by greg0ire** [comment]
Thanks @roukmoute !

**Pull request merged**